webui/react/package.json

```diff
@@ -38,6 +38,7 @@
     "@uiw/react-codemirror": "^4.20.2",
     "ansi-to-html": "^0.7.2",
     "antd": "^5.1.7",
+    "core-js": "3.32",
```

Comment:
i think polyfill should be handled by Vite, then we dont need to install new libs in dependencies and we can use structuredClone directly

Reply:
Does polyfill work if `structuredClone` is used directly without`core-js/actual/structured-clone`?